docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -365,12 +365,20 @@ When you generate a Bare Metal cluster configuration, the `TinkerbellTemplateCon
 Advanced users can override the default values set for `TinkerbellTemplateConfig`.
 They can also add their own [Tinkerbell actions](https://docs.tinkerbell.org/actions/action-architecture/) to make personalized modifications to EKS Anywhere nodes.
 
+A default `TinkerbellTemplateConfig` can be generated from cluster config template using eksctl:
+```bash
+eksctl anywhere generate tinkerbelltemplateconfig -f eksa-mgmt-cluster.yaml
+```
+
 The following shows three `TinkerbellTemplateConfig` examples that you can add to your cluster configuration file to override the values that EKS Anywhere sets: one for Ubuntu, one for RHEL and one for Bottlerocket.
 Most actions used differ for different operating systems.
 
 >**_NOTE:_** For the `stream-image` action, `DEST_DISK` points to the device representing the entire hard disk (for example, `/dev/sda`).
-For UEFI-enabled images, such as Ubuntu and RHEL 9, write actions use `DEST_DISK` to point to the second partition (for example, `/dev/sda2`), with the first being the EFI partition.
+For all the write actions, `DEST_DISK` must be set to point to the root partition.
+For Ubuntu, write actions use `DEST_DISK` to point to the second partition (for example, `/dev/sda2`), with the first being the EFI partition.
+For RHEL 8 and RHEL 9, `DEST_DISK` is partition 1 (for example, `/dev/sda1`).
```

Comment:
should this also be "pointed to" partition 1?

Reply:
changed to "points to"